addon/v1alpha1/0000_00_addon.open-cluster-management.io_clustermanagementaddons.crd.yaml

```diff
@@ -26,9 +26,9 @@ spec:
     schema:
       openAPIV3Schema:
         description: ClusterManagementAddOn represents the registration of an add-on
-          to the cluster manager. This resource allows the user to discover which
-          add-on is available for the cluster manager and also provides metadata information
-          about the add-on. This resource also provides a linkage to ManagedClusterAddOn,
+          to the cluster manager. This resource allows users to discover which add-ons
+          are available for the cluster manager and also provides metadata information
+          about these add-ons. This resource also provides a linkage to ManagedClusterAddOn;
           the name of the ClusterManagementAddOn resource will be used for the namespace-scoped
           ManagedClusterAddOn resource. ClusterManagementAddOn is a cluster-scoped
           resource.
```

Comment:
Suggestion: ClusterManagementAddOn registers add-ons
          to the cluster manager. This resource allows you to discover which add-ons
          are available for the cluster manager and provides metadata information
          about the add-ons. The ClusterManagementAddOn name is used for the namespace-scoped ManagedClusterAddOn resource. ClusterManagementAddOn is a cluster-scoped
          resource.

Reply:
Thanks! Replaced the suggestion content.